npm registry has 'brave' taken by another user, should use 'brave-browser' instead

**Did you search for similar issues before submitting this one?**
Yes 

**Describe the issue you encountered:**
When running `npm start` to run the browser locally, and start fails with: `code ELIFECYCLE` error message is: 

```

npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "start"
npm ERR! node v6.9.2
npm ERR! npm  v3.10.9
npm ERR! code ELIFECYCLE
npm ERR! brave@0.13.0 start: `node ./tools/start.js --user-data-dir=brave-development --debug=5858 --enable-logging --v=0 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck`
npm ERR! Exit status 7
npm ERR!
npm ERR! Failed at the brave@0.13.0 start script 'node ./tools/start.js --user-data-dir=brave-development --debug=5858 --enable-logging --v=0 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the brave package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node ./tools/start.js --user-data-dir=brave-development --debug=5858 --enable-logging --v=0 --enable-extension-activity-logging --enable-sandbox-logging --enable-dcheck
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs brave
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls brave
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     ~/repos/browser-laptop/npm-debug.log
```
(among other issues). However, `npm bugs brave` opens: https://github.com/davidjamesstone/brave/issues
`npm owner ls brave` yields: davidjamesstone <davidjamesstone@gmail.com>

These are not Brave's pages. The npm registry for `brave` does not point to what you expect it to point to: https://registry.npmjs.org/brave


**Expected behavior:**
Running terminal command redirect's to one of Github's bugs pages like: https://github.com/brave/browser-laptop

- Platform (Win7, 8, 10? macOS? Linux distro?):
macOS

- Brave Version (revision SHA):
Irrelevant

- Steps to reproduce:
    1.
    2.
    3.

- Screenshot if needed:

- Any related issues:

--- a/package.json
+++ b/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "brave",
+  "name": "brave-browser",
   "version": "0.13.0",
   "description": "Brave laptop and desktop browser",
   "main": "./app/index.js",

